ui/javascripts/app/routes.js

```diff
@@ -340,9 +340,6 @@ App.NodesShowRoute = App.BaseRoute.extend({
       },
       node: Ember.$.getJSON(formatUrl(consulHost + '/v1/internal/ui/node/' + params.name, dc.dc, token)).then(function(data) {
         return App.Node.create(data);
-      }),
-      nodes: Ember.$.getJSON(formatUrl(consulHost + '/v1/internal/ui/node/' + params.name, dc.dc, token)).then(function(data) {
```

Comment:
I'm concerned about this one because we're later setting it:

https://github.com/hashicorp/consul/blob/master/ui/javascripts/app/routes.js#L359-L364

That will currently be nil'd out. I assume if you test this and do:

1. View the nodes index page
2. Click a specific node

This will then be unnecessary, but did you try visiting a node directly? I would worry it wouldn't then populate the left column given this commit.

If even that works, we should likely remove the above linked set of this attribute.